cmd/print.go

```diff
@@ -34,6 +38,13 @@ func printMsg(msg message) {
 		msgStr = msg.String()
 	} else {
 		msgStr = msg.JSON()
+		if globalJSONLine && strings.ContainsRune(msgStr, '\n') {
+			// Reformat.
+			var dst bytes.Buffer
+			if err := json.Compact(&dst, []byte(msgStr)); err == nil {
+				msgStr = dst.String()
+			}
```

Comment:
Let's fatal or panic when there is an error here, to avoid hiding weird bugs.

Reply:
@vadmeste There are 87 implementations and if one returns bad JSON that will crash instead of just writing JSON (or whatever is returned) with newlines.